src/ovirtapi/index.js

```diff
@@ -153,8 +153,18 @@ const OvirtApi = {
   },
   getAllClusters ({ additional }: { additional: Array<string> }): Promise<Object> {
     assertLogin({ methodName: 'getAllClusters' })
-    const url = `${AppConfiguration.applicationContext}/api/clusters` +
-      (additional && additional.length > 0 ? `?follow=${additional.join(',')}` : '')
+
+    let follow
```

Comment:
Actually I don't like when variable have unknown value, thus, please, set it to empty string.

Reply:
That would be a wasted assignment.  I'll rearrange.